vividus-util/src/main/java/org/vividus/util/property/PropertyParser.java

```diff
@@ -43,6 +45,43 @@
                 .collect(toMap(e -> removeStart(e.getKey(), propertyPrefix), Entry::getValue));
     }
 
+    @Override
+    public Map<String, Object> getPropertyValuesTreeByPrefix(String propertyPrefix)
+    {
+        Map<String, Object> container = new HashMap<>();
+        getPropertyValuesByPrefix(propertyPrefix).forEach((k, v) -> putByPath(container, k, v));
+        return container;
+    }
+
+    @SuppressWarnings("unchecked")
+    private static void putByPath(Map<String, Object> container, String path, String value)
+    {
+        String[] paths = path.split("\\.");
```

Comment:
`org.apache.commons.lang3.StringUtils#split(java.lang.String)`

Reply:
changed to org.apache.commons.lang3.StringUtils#split(java.lang.String,char) ( space is default sep)